### PR TITLE
Cleaning up linting rules and updating code to match

### PR DESCRIPTION
**Pull request by blarghmatey**

### CODE REVIEW

File: src/bilder/components/caddy/steps.py [hunk 1]
```diff
@@ -12,7 +13,7 @@ def install_caddy(caddy_config: CaddyConfig, state=None, host=None):
         caddy_user = "caddy"
         server.user(
             name="Create system user for Caddy",
-            user="caddy",
+            user=caddy_user,
```
Comment (shaidar) [inline src/bilder/components/caddy/steps.py:16]: Why the change?

File: src/bilder/components/caddy/steps.py [hunk 2]
```diff
@@ -62,9 +63,9 @@ def install_caddy(caddy_config: CaddyConfig, state=None, host=None):
         )
         apt.repo(
             name="Set up Caddy APT repository",
-            src="deb https://dl.cloudsmith.io/public/caddy/stable/deb/debian any-version main",
+            src="deb https://dl.cloudsmith.io/public/caddy/stable/deb/debian any-version main",  # noqa: E501
             present=True,
-            filename="caddy",
+            filename="caddy.list",
```
Comment (shaidar) [inline src/bilder/components/caddy/steps.py:68]: ?

File: src/bilder/components/caddy/steps.py (hunk 1 above)
Comment (blarghmatey) [reply src/bilder/components/caddy/steps.py:16]: To bring it in line with the other uses of that variable and to cut down on overuse of the string constant.

File: src/bilder/components/caddy/steps.py (hunk 2 above)
Comment (blarghmatey) [reply src/bilder/components/caddy/steps.py:68]: Just fixing the file name to match the standard `.list` suffix for apt repositories.

**Comment by shaidar** [comment]
👍
